src/resources/views/crud/list.blade.php

```diff
@@ -50,6 +50,7 @@
         <table
           id="crudTable"
           class="bg-white table table-striped table-hover nowrap rounded shadow-xs border-xs mt-2"
+          data-responsive="{{ $crud->getOperationSetting('responsive') ?? 0 }}"
```

Comment:
I assume you meant `responsiveTable` here, not `responsive`? That's what the config name is.

```suggestion
          data-responsive="{{ $crud->getOperationSetting('responsiveTable') ?? 0 }}"
```

Reply:
I'm asking for this because for me, without it... when the table should have been responsive... it's not. It spans the full width of the container.

Reply:
This is separate from the arrows issue. But it's even more important, because without it... I can't really see all the buttons on a line, in some tables.

Reply:
Fixed 🙌 This was why I was having some issues 🤦‍♂️